configDiffer/diff.py

```
@@ -0,0 +1,55 @@
+    begin: int | None
+    end: int | None
+    line_diffs: dict[int, LineDiff]
+
+
+def diff_file(file_path1: Path, file_path2: Path) -> ParagraphDiff:
+    file1 = open(file_path1, "r").readlines()
+    file2 = open(file_path2, "r").readlines()
+    line_diffs = {}
+    start = None
+    end = None
+    for i, (line1, line2) in enumerate(zip(file1, file2)):
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)